port/include/input.h

```diff
@@ -232,6 +274,100 @@ void inputMouseSetSpeed(f32 x, f32 y);
 s32 inputMouseIsEnabled(void);
 void inputMouseEnable(s32 enabled);
 
+// Gyro Controller Initialization & Management
+void inputHandleGyroController(s32 cidx);
+void closeGyroController(s32 cidx);
+s32 inputGyroIsEnabled(s32 cidx);
+void inputGyroEnable(s32 cidx, s32 enabled);
+
+// Gyro Vertical/Horizon Mixer Adjustment
+f32 inputGetGyroVHMixer(s32 cidx);
+void inputSetGyroVHMixer(s32 cidx, f32 value);
+
+// Raw Gyro Movement Retrieval
+void inputGyroGetRawDelta(s32 cidx, s32* dx, s32* dy, s32* dz);
+
+// Update Gyro Data
+void inputUpdateGyro(s32 cidx);
+
+// Scaled Gyro Movement Retrieval
+void inputGyroGetScaledDelta(s32 cidx, f32* dx, f32* dy, f32* dz);
+
+// Gyro Crosshair Movement Retrieval & Adjustment
+void inputGyroGetScaledDeltaCrosshair(s32 cidx, f32* dx, f32* dy);
+void inputGyroGetCrosshairDelta(s32 cidx, f32* dx, f32* dy);
+
+// Gyro FreeLook Sensitivity Management
+void inputGyroGetSpeed(s32 cidx, f32* x, f32* y);
+void inputGyroSetSpeed(s32 cidx, f32 x, f32 y);
+f32 inputGyroGetSpeedX(s32 cidx);
+void inputGyroSetSpeedX(s32 cidx, f32 x);
+f32 inputGyroGetSpeedY(s32 cidx);
+void inputGyroSetSpeedY(s32 cidx, f32 y);
```

Comment:
What's the point of separate functions for X and Y if you already have `GetSpeed`? Just do it the same way as `MouseGetSpeed`, the gyro API is already massive enough as it is.